packages/mobile/src/account/Account.tsx

```diff
@@ -88,8 +90,12 @@ export class Account extends React.Component<Props, State> {
     navigate(Screens.Profile)
   }
 
-  goToBackupScreen() {
-    navigate(Screens.BackupIntroduction)
+  goToBackupScreen = () => {
+    if (this.props.backupCompleted) {
+      navigateProtected(Screens.BackupIntroduction)
```

Comment:
I see, that `Screens.BackupIntroduction` has here two different purposes depend on `backupCompleted`. Do you think we can split this screen into two and keep `Screens.BackupIntroduction` only for Introduction?

Reply:
My thought is that while yes it could be two, its better to draw a line somewhere as to what we will and will not change in the redesign. for me that includes not creating new screens if any exisiting one is hooked up like we need. 

Without this Line I would have also renamed this component, and the i18n key file at leas, maybe more.